packages/http-client-python/emitter/src/run-python3.ts

```diff
@@ -11,7 +11,7 @@ import { patchPythonPath } from "./system-requirements.js";
 
 export async function runPython3(...args: string[]) {
   const command = await patchPythonPath(["python", ...args], {
-    version: ">=3.8",
+    version: ">=3.9",
```

Comment:
Technically speaking, using 3.8+ to generate and generating 3.8+ are two different things. For instance, we could technically have a generator written in Python 2.7, that generates Python 3.12 code :). In other words, the fact that our generator is written in Python is an implementation detail.

That being said, it's fair to ask people to have 3.8+, because it allows us as well when we writee the generator to use more recent syntax.